### ISSUE TITLE
Accordion Menu methods do not work with Responsive Navigation

### ISSUE THREAD

**Issue by scottsawyer** [zurb/foundation-sites#11769]
## What should happen?
When using the Responsive Menu, all of the plugin methods for the different menu types referenced should work.  

Example:
Accordion Menu has the following method:
`$('#element').foundation('toggle', $target);`
However, this and all other accordion menu methods do not work when using accordion with responsive menu.  

...

## What happens instead?
> Uncaught ReferenceError: We're sorry, 'toggle' is not an available method for e.
...

## Possible Solution
Best outcome would be that I have made a conceptual mistake and there is actually a way to trigger the submenu toggle with responsive menu.  

Second best would be to enable all of the responsive menu plugin methods.  

...

## Test Case and/or Steps to Reproduce (for bugs)

Test Case:  https://codepen.io/anon/pen/EzBXKy

In the console, run $('#my-menu').foundation('toggle', $target);

Set up a responsive menu with:
`data-responsive-menu="drilldown medium-accordion"`

Then try running any of the accordion menu methods.

## Context
I am building a map of the US with a Responsive Menu of states and nested cities.  I want users to be able to click a state on the map to open the corresponding submenu.  Essentially, trigger a submenu via an outside trigger.  Due to space constraints, I am using Drilldown menu on small and Accordion Menu on medium and up.  
...

## Your Environment

- Foundation version(s) used: 6.5.3
- Browser(s) name and version(s): 74.0.3729.157
- Device, Operating System and version: ubuntu 16.04 
- Link to your project:  https://codepen.io/anon/pen/EzBXKy

## Checklist

- [X ] I have read and follow the CONTRIBUTING.md document.
- [X ] There are no other issues similar to this one.
- [ X] The issue title and template are correctly filled.

**Comment by DanielRuf** [comment]
Hi @scottsawyer,

At the first sight `Drilldown` has no `toggle` event listener.

https://github.com/zurb/foundation-sites/blob/develop/js/foundation.drilldown.js

**Comment by scottsawyer** [comment]
This is true, but Accordion menu should.
https://foundation.zurb.com/sites/docs/accordion-menu.html#js-functions

**Comment by DanielRuf** [comment]
We have multiple issues here in Foundation Sites.

1) the minified version does not use the original class names so it sets `data-e` instead of `data-responsive-menu`

Which results in
> Uncaught ReferenceError: We're sorry, 'toggle' is not an available method for e.

Using the unminified version of Foundation Sites results in
> Uncaught ReferenceError: We're sorry, 'toggle' is not an available method for ResponsiveMenu.

Which is partly correct. It seems only the first plugin is checked and we can not use the other plugins which are applied as data attributes.

As this has no `toggle` method. See https://github.com/zurb/foundation-sites/blob/develop-v6.5/js/foundation.responsiveMenu.js

**Comment by DanielRuf** [comment]
Related:

https://github.com/zurb/foundation-sites/issues/11743
https://github.com/zurb/foundation-sites/pull/11764/files

**Comment by DanielRuf** [comment]
Minified results in
```
data-responsive-menu="drilldown medium-accordion"
data-multi-open="false"
data-n="85njhk-n"
data-mutate="kazl0v-responsive-menu"
data-e="65lzfp-e"
```

Unminified results in
```
data-responsive-menu="drilldown medium-accordion"
data-multi-open="false"
data-accordion-menu="1neo7n-accordion-menu"
data-mutate="abg98g-responsive-menu"
```

**Comment by scottsawyer** [comment]
Do you think that pull will address this problem?  Is it only a problem in the unminified version?  Would I need the unminified version of all the plugins or just the responsiveMenu.js?

**Comment by DanielRuf** [comment]
> Using the unminified version of Foundation Sites results in
> 
> > Uncaught ReferenceError: We're sorry, 'toggle' is not an available method for ResponsiveMenu.
> 
> 1. Which is partly correct. It seems only the first plugin is checked and we can not use the other plugins which are applied as data attributes.

There are probably more issues.

**Comment by scottsawyer** [comment]
Yikes, that is bad news for my project.  

Are you saying that the pull request you referenced in https://github.com/zurb/foundation-sites/issues/11769#issuecomment-500033992 does not fix the underlying problem?

**Comment by DanielRuf** [comment]
Exactly. Can you test if all previous versions are also affected (after 6.4.3 probably) so we can do some bisecting to find the commit which caused this?

This would be very helpful.

**Comment by scottsawyer** [comment]
Well, I just switched to 6.4.3 in my pen and running:
`$('#my-nav').foundation('showAll');`
returns:
`"We're sorry, 'showAll' is not an available method for e."`
So, I am wonder if it ever worked.

**Comment by BreathingFlesh** [comment]
This still seems to be an issue. Has anybody got a solution for this?
